### PR TITLE
build: nix flake for a dev shell

### PR DESCRIPTION
**Pull request by aljazerzen**

### CODE REVIEW

File: .config/vscode-recommended/launch.json
Comment (max-sixty) [inline .config/vscode-recommended/launch.json:1]: Are we adding these too?

**Review by max-sixty — APPROVED** [review]
Looks good!

File: .config/vscode-recommended/launch.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "lldb",
+      "request": "launch",
+      "name": "prqlc _a.prql",
```
Comment (max-sixty) [inline .config/vscode-recommended/launch.json:10]: What is this?

Comment (aljazerzen) [reply .config/vscode-recommended/launch.json:10]: A vscode configuration for running `prqlc debug semantics _a.prql`, with lldb debugger attached, so you can set breakpoints and step trough the program.

File: .config/vscode-recommended/launch.json
Comment (aljazerzen) [reply .config/vscode-recommended/launch.json:1]: These are in the `.config/vscode-recommended/`.

I had an "init" script that copies them into `.vscode/`

File: .config/vscode-recommended/launch.json (hunk 1 above)
Comment (max-sixty) [reply .config/vscode-recommended/launch.json:10]: But what's `_a.prql`?

Comment (aljazerzen) [reply .config/vscode-recommended/launch.json:10]: A local file containing the query you are currently working on.

Comment (max-sixty) [reply .config/vscode-recommended/launch.json:10]: I see, OK

Comment (aljazerzen) [reply .config/vscode-recommended/launch.json:10]: Mine usually has a bunch of different cases commented out, so I can enable them quickly.

File: .config/vscode-recommended/launch.json
Comment (max-sixty) [reply .config/vscode-recommended/launch.json:1]: OK!

**Pull request merged**